anyway/infographics_utils.py

```diff
@@ -1404,9 +1407,10 @@ def get_query(table_obj, filters, start_time, end_time):
 
 
 def get_accidents_stats(
-    table_obj, filters=None, group_by=None, count=None, start_time=None, end_time=None
+    table_obj, filters={}, group_by=None, count=None, start_time=None, end_time=None
```

Comment:
Are you sure filters will never by None? If yes, then it is worthwhile to set its type, so lint will catch an issue.